source/simulator/solver_schemes.cc

```diff
@@ -291,8 +291,9 @@ namespace aspect
         const double relative_stokes_residual =
           assemble_and_solve_stokes(nonlinear_iteration == 0, &initial_stokes_residual);
 
-        pcout << "      Relative Stokes residual after nonlinear iteration " << nonlinear_iteration+1
+        pcout << "      Nonlinear residual (Stokes system only) after nonlinear iteration " << nonlinear_iteration+1
```

Comment:
I think it is better to name them all `Relative nonlinear residual`, like you suggested in #1964.